.github/workflows/integration_tests.yml

```
@@ -261,7 +261,7 @@ jobs:
         #if: github.event_name == 'schedule' || inputs.target == 'nightly' || inputs.target == 'anyon'
         if: inputs.target == 'anyon'
         run: |
-          curl -X POST -H "Content Type: application/json" -d '{ "username":"${{ secrets.BACKEND_LOGIN_USERNAME }}","password":"${{ secrets.BACKEND_LOGIN_PASSWORD }}" }' https://api.anyon.cloud/login > credentials.json
+          curl -X POST --user "${{ secrets.BACKEND_LOGIN_USERNAME }}:${{ secrets.BACKEND_LOGIN_PASSWORD }}"  -H "Content-Type: application/json" https://api.anyon.cloud:5000/login > credentials.json
```

Could we please put "ANYON" in these secret names so that it is easier for us to tell the backends apart?
```suggestion
          curl -X POST --user "${{ secrets.ANYON_USERNAME }}:${{ secrets.ANYON_PASSWORD }}"  -H "Content-Type: application/json" https://api.anyon.cloud:5000/login > credentials.json
```